googleapi/googleapi.go

```diff
@@ -259,6 +259,20 @@ func ChunkSize(size int) MediaOption {
 	return chunkSizeOption(size)
 }
 
+type chunkTransferTimeoutOption time.Duration
+
+func (cd chunkTransferTimeoutOption) setOptions(o *MediaOptions) {
+	o.ChunkTransferTimeout = time.Duration(cd)
+}
+
+// ChunkTransferTimeout returns a MediaOption which sets a per-chunk
+// transfer timeout. If a single chunk has been attempting to upload for longer than
+// this time then the old req got canceled and retried.
+// The default value is 0s. This ensures that existing clients won't experience any changes in behavior.
```

Comment:
I would drop the sentence about existing clients; instead something like:

`// The default is no timeout for the request.`